mitodl/unified-ecommerce
# Remove Traefik support

### What are the relevant tickets?

n/a

### Description (What does it do?)

Initially, we were targeting both APISIX and Traefik as the API gateway for the system. We've since decided on APISIX as the gateway, so this removes the Traefik support. 

### How can this be tested?

If you're running this and using Traefik as the gateway, it will break, and you will need to follow the instructions to set the gateway back up. (This will not change anything with the app itself or its data.)

Otherwise, the app should work as expected using APISIX as the gateway, and the documentation should be fairly clear.  

### Additional Context

The app runs APISIX by default - maybe we want to split this out in future (as part of a larger "OL apps shared services" environment that'd also include Keycloak, etc).
